editor/src/components/canvas/dom-walker.ts

```diff
@@ -436,8 +437,8 @@ export function useDomWalker(props: CanvasContainerProps): React.Ref<HTMLDivElem
             canvasRootPath as InstancePath,
             left('Storyboard'),
             {},
-            null,
-            null,
+            { x: 0, y: 0, width: 0, height: 0 } as CanvasRectangle,
```

Comment:
x and y are correct - they are indeed 0 for the canvas root. width and height could be infinity or NaN, but 0 felt appropriate as well – no code should rely on it anyways